libs/core/src/test/java/org/elasticsearch/core/ReleasablesTests.java

```diff
@@ -74,4 +73,39 @@ public String toString() {
                 .anyMatch(ste -> ste.toString().contains("CloserWithIdentifiableMethodNames.closeMethod1"))
         );
     }
+
+    public void testWrap() {
+        final var count = new AtomicInteger(0);
+        final Releasable releasable = new Releasable() {
+            @Override
+            public void close() {
+                count.incrementAndGet();
+            }
+
+            @Override
+            public String toString() {
+                return "increment count";
+            }
+        };
+
+        final var wrapVarArgs = Releasables.wrap(releasable, releasable);
+        assertEquals("wrapped[increment count, increment count]", wrapVarArgs.toString());
+        wrapVarArgs.close();
+        assertEquals(2, count.get());
+
+        final var wrapIterable = Releasables.wrap(new Iterable<>() {
```

Comment:
List implements Iterable, so this could just be `List.of(releasable, releasable, releasable)`?

Reply:
Not without losing the ability to override its `toString()` tho.